docs/explanation/gno-tooling/cli/common-params.md

```diff
@@ -0,0 +1,16 @@
+---
+id: gno-tooling-common-params
+---
+
+# Common Parameters
+
+Below is a list of common parameters.
+
+| Name                      | Description                          | Default                                  |
+| ------------------------- | ------------------------------------ | ---------------------------------------- |
+| `home`                    | The home directory.                  | `GNO_HOME` or the user's home directory. |
```

Comment:
Can we check with latest changes if it's still appropriate?

cc @thehowl @gfanton

Reply:
ping @thehowl @gfanton

Reply:
I've thought about this a bit, and I think we should probably just drop the common params doc, since it has a huge risk of going outdated. I am considering doing the same for all "tools" in the tooling section (their `--help` outputs are the most up-to-date documentation anyways)

[00cbe6d](https://github.com/gnolang/gno/pull/1046/commits/00cbe6ddf85d552dcb36c3c6ab4abc4e52f32209)